packages/libs/error-stack/src/report.rs

```diff
@@ -617,10 +665,22 @@ impl<Context> fmt::Debug for Report<Context> {
     }
 }
 
+#[cfg(feature = "std")]
+impl<Context> std::process::Termination for Report<Context>
+where
+    Context: fmt::Debug,
+{
+    fn report(self) -> std::process::ExitCode {
+        std::process::ExitCode::from(self.inner.exit_code.unwrap_or(1))
+    }
+}
+
 pub struct ReportImpl {
     pub(super) frame: Frame,
     #[cfg(all(nightly, feature = "std"))]
     backtrace: Option<Backtrace>,
     #[cfg(feature = "spantrace")]
     span_trace: Option<SpanTrace>,
+    #[cfg(feature = "std")]
+    exit_code: Option<u8>,
```

Comment:
Instead of storing the exit code inside the report, it's probably more consistent with treating it as an attachment, see below.